src/test/miner_tests.cpp

```diff
@@ -57,12 +57,13 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
     CMutableTransaction tx,tx2;
     CScript script;
     uint256 hash;
+    CBlockIndex *pindexPrev = NULL;
 
     LOCK(cs_main);
     Checkpoints::fEnabled = false;
 
     // Simple block creation, nothing special yet:
-    BOOST_CHECK(pblocktemplate = CreateNewBlock(scriptPubKey));
+    BOOST_CHECK(pblocktemplate = CreateNewBlock(scriptPubKey, pindexPrev, Params()));
```

Comment:
Thank you for doing this, but if you do it, you should do it right. That is, with 
```
const CChainParams& params = Params(CBaseChainParams::MAIN);
```
at the beginning of the test case as in https://github.com/bitcoin/bitcoin/pull/5970/files#diff-5c64500485fda76388a86c95c0059585R53